scanner/matcher.go

```diff
@@ -127,3 +131,94 @@ func (m MatchIssuerRegex) CertificateMatches(c *x509.Certificate) bool {
 func (m MatchIssuerRegex) PrecertificateMatches(p *ct.Precertificate) bool {
 	return m.PrecertificateIssuerRegex.FindStringIndex(p.TBSCertificate.Issuer.CommonName) != nil
 }
+
+// RawMatcher describes how to match log entries, based on the raw log entry information;
```

Comment:
Not sure about 'raw'. What about making it LeafMatcher? and changing comment to "based on the log LeafEntry"? Method could then maybe just be Matches() but this might be clearer as is.

Reply:
Done.